nix_update/__init__.py

```diff
@@ -114,6 +114,12 @@ def parse_args(args: list[str]) -> Options:
         help="Attribute of a subpackage that nix-update should try to update hashes for",
         default=None,
     )
+    parser.add_argument(
+        "--extra-flags",
```

Comment:
I think I would prefer to not shell split and limit it to option for now:

```
    parser.add_argument(
        "--option",
        help="Nix option to set",
        action="append",
        nargs=2,
        metavar=("name", "value"),
        default=[],
    )
```

Other flags would likely fail because we have different commands that could run.

Reply:
Applied this change and tested with `nix-update --option --extra-experimental-features pipe-operators`